community-edition/generate_script/hcce.yam

```diff
@@ -338,13 +338,13 @@ data:
 
     [ret."Elixir.RetWeb.Plugs.AddCSP"]
     child_src = ""
-    connect_src = "wss://*.stream.<DOMAIN>:4443"
+    connect_src = "wss://*.stream.<DOMAIN>:4443 demo.hubsfoundation.org *.demo.hubsfoundation.org"
```

Comment:
Should we remove `wss://*.stream.<DOMAIN>:4443` because reticulum already adds it?